tests/integration/test_docker.py

```diff
@@ -112,51 +102,15 @@ def test_docker_run_file_upload_to_item(session, girder_client, test_item):
 
 
 @pytest.mark.docker
-def test_docker_run_girder_file_to_named_pipe_on_temp_vol(session, test_file, test_file_in_girder):
-    """
-    This is a simplified version of test_docker_run_girder_file_to_named_pipe
-    it uses the TemporaryVolume, rather than having to setup the volumes
-    'manually', this is the approach we should encourage.
```

Comment:
Was there a reason this doc string was removed?

Reply:
Kind of an unintended consequence of collapsing the two tests into one via `parametrize`... I don't fully understand the comment, do you think it's important to have it here inline?

Reply:
If you don't understand it then its best left removed :smile: